.github/ISSUE_TEMPLATE/eip-tracker.yaml

```
@@ -0,0 +1,81 @@
+name: "EIP Implementation Tracker"
+description: "Track specification and testing progress for an EIP"
+title: "EIP ${{ form.eip_number }} Progress Tracker"
+labels:
+  - A-spec-specs
+  - A-spec-tests
+  - C-eip
+  - C-test
+
+body:
+  - type: input
+    id: eip_number
+    attributes:
+      label: "EIP Number"
+      description: "Enter the EIP number (digits only)."
+      placeholder: "e.g., 8024"
+    validations:
+      required: true
+
+  - type: input
+    id: eip_title
+    attributes:
+      label: "EIP Title"
+      description: "Copy the title from the EIP."
+      placeholder: "e.g., Backwards compatible SWAPN, DUPN, EXCHANGE"
+    validations:
+      required: true
+  
+  - type: dropdown
+    id: fork
+    attributes:
+      label: "Fork"
+      description: |
+        Specify the target fork **only if the EIP has reached the CFI stage**.
+        More info: https://eips.ethereum.org/EIPS/eip-7723#considered-for-inclusion
+      options:
+        - TBD
+        - amsterdam
+        - bogota
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        ## [EIP-${{ form.eip_number }}](https://eips.ethereum.org/EIPS/eip-${{ form.eip_number }})
+
+        ### Target Fork
+
+        Fork **${{ form.fork }}**
+
+        ### Instructions
+        
+        - [ ] Assign issue to EIP specification and testing owner(s).
+
+        > [!IMPORTANT]
+        > A specifications specialist and a testing specialist should ideally share ownership of the EIP.
+
+        - [ ] Add the issue to the target fork milestone if applicable (i.e., the EIP is at least in the (CFI stage)[https://eips.ethereum.org/EIPS/eip-7723#considered-for-inclusion]).
```

broken link styling I think

```suggestion
        - [ ] Add the issue to the target fork milestone if applicable (i.e., the EIP is at least in the [CFI stage](https://eips.ethereum.org/EIPS/eip-7723#considered-for-inclusion)).
```